cluster/gke/util.sh

```diff
@@ -68,12 +69,28 @@ function test-build-release() {
 
 # Verify needed binaries exist.
 function verify-prereqs() {
-  echo "... in verify-prereqs()" >&2
-
-  ${GCLOUD} preview --help >/dev/null || {
-    echo "Either the GCLOUD environment variable is wrong, or the 'preview' component"
-    echo "is not installed. (Fix with 'gcloud components update preview')"
-  }
+  local cmd
+  for cmd in gcloud; do
+    if ! which "${cmd}" >/dev/null; then
+      echo "Can't find ${cmd} in PATH.  Do you wish to install the Google Cloud SDK? [Y/n]"
```

Comment:
This prompt won't be suppressed by the KUBE_PROMPT_FOR_UPDATE var above. Was that intentional?